### ISSUE TITLE
Need an alternative to Insert key on OSX

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kometbomb/klystrack#37]
```
So I've been playing around with Klystrack and am rather impressed by it. 
Except for one thing: I could not immediately figure out how to extend the 
length of a pattern. There's no quick GUI button to do it (the best I could 
find was Edit > Pattern > Expand), and I couldn't figure out the keyboard 
shortcut despite my best guesses. So I looked in the help file...

Alt+Insert. Ah, of course.

Pressing a key that doesn't exist on my laptop is definitely an interesting 
approach, but hardly intuitive. And although the "Help" key on desktop Macs 
used to be equivalent to Insert... it no longer exists on Apple's most recent 
keyboards, instead having been replaced by a "Fn" key.

In short: There needs to be a secondary shortcut for laptops (and desktop 
keyboards, even!) that don't have an Insert key.
```

Original issue reported on code.google.com by `codeman38` on 10 Jan 2011 at 6:30
